api/cluster/resource/templater.go

```diff
@@ -910,3 +931,11 @@ func (t *InferenceServiceTemplater) applyDefaults(service *models.Service) {
 		}
 	}
 }
+
+func ParseEnvVars(envVars []corev1.EnvVar) models.EnvVars {
+	var parsedEnvVars models.EnvVars
+	for _, envVar := range envVars {
+		parsedEnvVars = append(parsedEnvVars, models.EnvVar{Name: envVar.Name, Value: envVar.Value})
+	}
+	return parsedEnvVars
```

Comment:
This helper function converts the default env vars from the `corev1.EnvVar` type (from the k8s library) to the `models.EnvVar` type (defined and used only in the Merlin API server) so that the existing helper method to merge env vars `models.MergeEnvVars` can be easily reused.